### ISSUE TITLE
Document Browser compatibility of Launch DApps

### ISSUE THREAD

**Issue by Tartuffo** [Agoric/Agoric#5583]
At Launch, unless we implement a different Wallet Connect protocol (#5506), we will only support Firefox and Chromium-based browsers.  We should clearly document that on our website, along with any necessary configuration settings such disabling Shields Up in Brave.

If feasible, we should also detect when a Chromium browser has settings that preclude the DAPpps from running correctly, e.g. Brave with Shields Up.

**Comment by otoole-brendan** [comment]
I've included a note at top of PSM page that notifies users to turn off their Brave shields before continuing with PSM dapp

**Comment by dckc** [comment]
I just got a problem report from a colleague:
> when i press Connect Kepler (after accepting terms) nothing seems to happen. iphone firefox.

I didn't realize there was such a thing as keplr for firefox, let alone for iphone. But there is: https://addons.mozilla.org/en-US/firefox/addon/keplr/

@otoole-brendan do we document supported browsers in https://docs.inter.trade/ ? I'm struggling to find it.

**Comment by turadg** [comment]
Relevant:

 -  https://github.com/Agoric/dapp-psm/issues/30

**Comment by otoole-brendan** [comment]
Re-prioritizing this for Vaults release - we need to be able to say what browsers we support.

**Comment by turadg** [comment]
Once this is done we need to QA the dapps on each browser and common configurations. For example, Brave with Shields or Firefox with Tracking Prevention. Whatever diagnostic signals we use will have to be robust to browser changes, such as Chrome's First-Party Sets rollout that broke the wallet bridge: https://github.com/Agoric/dapp-psm/issues/69

**Comment by otoole-brendan** [comment]
I have documented supported browsers on Gitbook: https://docs.inter.trade/user-how-to/supported-browsers

**Comment by otoole-brendan** [comment]
I'd like to do a bit more testing on Firefox before closing ticket though. 
@turadg I see diagnostic signals/tools for UX being a separate issue (I'm happy to raise). I'd like to continue the discussion to know more about what's possible on that front though - will take it to Slack or use new issue.

**Comment by otoole-brendan** [comment]
Docs updated. Closing for now - Edge testing required to change - not subject to this ticket - will create another if needed.
